.github/workflows/ciab.yaml

```diff
@@ -229,11 +229,6 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           path: ${{ github.workspace }}/dist/
-      - name: Cache Perl modules
-        uses: actions/cache@v2
-        with:
-          path: ${{ github.workspace }}/infrastructure/cdn-in-a-box/traffic_ops/perl-cache/centos-${{ env.RHEL_VERSION }}/local
```

Comment:
Nit: The `infrastructure/cdn-in-a-box/traffic_ops/perl-cache/centos-7/local/.gitkeep` and `infrastructure/cdn-in-a-box/traffic_ops/perl-cache/centos-7/local/.gitkeep` files can also be removed.

Reply:
I thought I got those already... done